nih2mne/utilities/anonymize_meg.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import numpy
+import os
+import shutil
+import sys
+from pathlib import Path
+
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.dirname(SCRIPT_DIR))
+sys.path.append(os.path.dirname(os.path.dirname(SCRIPT_DIR)))
+
+from make_meg_bids import anonymize_meg
```

Comment:
Should this be 
` from ..make_meg_bids import anonymize_meg`

Reply:
@jstout211 Yep, it's a relative import problem:

```python
Traceback (most recent call last):
  File "/home/earlea/repo/nih_to_mne/nih2mne/utilities/anonymize_meg.py", line 12, in <module>
    from ..make_meg_bids import anonymize_meg
ImportError: attempted relative import with no known parent package
```